calico_node/filesystem/etc/rc.local

```diff
@@ -61,8 +61,7 @@ case "$CALICO_NETWORKING_BACKEND" in
 		confd -onetime -backend=k8s -confdir=/etc/calico/confd -log-level=debug -keep-stage-file >/felix-startup-1.log 2>&1 || true
 		confd -onetime -backend=k8s -confdir=/etc/calico/confd -log-level=debug -keep-stage-file >/felix-startup-2.log 2>&1 || true
 	else
-		# Use ETCD_ENDPOINTS in preferences to ETCD_AUTHORITY
```

Comment:
This change isn't necessary - I don't think we are fully retiring ETCD_SCHEME and ETCD_AUTHORITY at this point - just that we don't want to advertise it in the CLI output.